08LessonEight/homework/css/exercise-06-style.css

```diff
@@ -1 +1,62 @@
-/*Add CSS rules*/
\ No newline at end of file
+/*Add CSS rules*/
+body {
+  width: 1000px;
+  margin: 0 auto;
+  padding: 40px 0;
+  font-family: "Arial", sans-serif;
+  font-size: 12px;
+  line-height: 16px;
+}
+section {
+
+}
+.clearfix: after {
+  content: "";
+  clear: both;
+  display: block;
+}
+div {
+  float: left;
+  width: 293px;
+  margin-bottom: 20px;
+}
+section div:nth-child(2) {
+  margin: 0 20px;
+}
+h3 {
+  margin-bottom: 20px;
+}
+p {
+  margin: 0 0 40px 0;
+  font-weight: normal;
+}
+a {
+  text-transform: uppercase;
+  text-decoration: none;
+  padding: 10px;
+  border: 1px solid lightgrey;
+  border-right: 3px solid red;
+  color: #000;
+}
+
+#truck .image {
```

Comment:
Noticed how your `#truck .image`, `#plane .image`, and `#train .image` have the same properties. How can you improve that and reduce redundancy?